armi/materials/zr.py

```diff
@@ -130,7 +130,7 @@ def linearExpansionPercent(self, Tk=None, Tc=None):
         Tk = getTk(Tc, Tk)
         self.checkPropertyTempRange("linear expansion percent", Tk)
 
-        if Tk >= 291.62 and Tk < 1137:
```

Comment:
This reverted a change that was needed to change the blueprints which also were not needed.